extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -71,4 +71,30 @@ amp-carousel .amp-carousel-button.amp-disabled {
   animation: none;
   opacity: 0;
   visibility: hidden;
-}
\ No newline at end of file
+}
+
+/* Begin SlideScroll */
+.-amp-slidescroll {
+  /* remove the gap between inline(-block) elements */
+  font: 0/0 a;
```

Comment:
removed it for now - there might be a small (invisible gap :)) we can fix it by removing text nodes i guess. Will do that in one of the following PRs